migrate/sqlMigrator/sqlMigrator_test.go

```diff
@@ -0,0 +1,183 @@
+	dstore := connectToDatastore()
+
+	// Generate projects
+	for i := 0; i < numProjectsGen; i++ {
+		proj, ttpl, stpl := generateProject(i)
+		err := dstore.CreateProject(proj, *ttpl, *stpl)
+		if err != nil {
+
+		}
+
+		// Populate projects with random number of accounts
+		accounts := generateAccounts(proj.ID)
+
+		for _, account := range *accounts {
+			err := dstore.InsertAccount(account)
+			if err != nil {
+				fmt.Println("Error: could not insert account into project", err)
+			}
+		}
+
+		// Populate projects with transaction executions
+		for _, exec := range *generateTransactionExecutions(proj) {
+			err = dstore.InsertTransactionExecution(&exec)
+			if err != nil {
+
+			}
+		}
+
+		// Populate projects with script executions
+		for _, exec := range *generateScriptExecutions(proj) {
+			err = dstore.InsertScriptExecution(&exec)
+			if err != nil {
+
+			}
+		}
+	}
+}
+
+func generateProject(projectGenCount int) (*model.InternalProject, *[]*model.TransactionTemplate, *[]*model.ScriptTemplate) {
+	proj := &model.InternalProject{
+		ID:                        uuid.New(),
+		UserID:                    uuid.New(),
+		Secret:                    uuid.New(),
+		PublicID:                  uuid.New(),
+		ParentID:                  nil,
+		Title:                     "Project number " + strconv.Itoa(projectGenCount),
+		Description:               "",
+		Readme:                    "",
+		Seed:                      0,
+		TransactionCount:          0,
+		TransactionExecutionCount: rand.Intn(5),
+		TransactionTemplateCount:  rand.Intn(5),
+		ScriptTemplateCount:       rand.Intn(5),
+		Persist:                   true,
+		CreatedAt:                 time.Time{},
+		UpdatedAt:                 time.Time{},
+		Version:                   build.Version(),
+	}
+
+	// Populate project with transaction templates
+	var ttpls []*model.TransactionTemplate
+	for i := 0; i < proj.TransactionTemplateCount; i++ {
+		ttpl := model.TransactionTemplate{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			Title:  "Transaction Template " + strconv.Itoa(i),
+			Index:  i,
+			Script: "Test script",
+		}
+
+		ttpls = append(ttpls, &ttpl)
+	}
+
+	// Populate project with script templates
+	var stpls []*model.ScriptTemplate
+	for i := 0; i < proj.ScriptTemplateCount; i++ {
+		stpl := model.ScriptTemplate{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			Title:  "Script Template " + strconv.Itoa(i),
+			Index:  i,
+			Script: "Test script",
+		}
+
+		stpls = append(stpls, &stpl)
+	}
+
+	return proj, &ttpls, &stpls
+}
+
+// generateAccounts generates a random number of accounts up to 10
+func generateAccounts(projID uuid.UUID) *[]*model.InternalAccount {
+	var accounts []*model.InternalAccount
+
+	for i := 0; i < rand.Intn(10); i++ {
+		// Generate address
+		bn := make([]byte, 8)
+		binary.BigEndian.PutUint32(bn, uint32(i))
+		addr := model.NewAddressFromBytes(bn)
```

Comment:
All accounts in the projects are just "0x01", "0x02", "0x03", "0x04", so just replicate that in tests